zeppelin-web/src/app/tabledata/tabledata.js

```diff
@@ -1,3 +1,7 @@
+/*
+ * @Author: curlsli@tencent.com
+ * @LastEditors: curlsli@tencent.com
```

Comment:
in ASF, we discourage edit or author tag. commit log already shows who did the work

Reply:
Thanks @felixcheung ，the comment has been deleted.